docs/run/primitives.mdx

```diff
@@ -6,68 +6,88 @@ description: Introduction to primitives in Qiskit and Qiskit Runtime, and an exp
-  * The `qiskit.primitives` module enables the development of primitive-style quantum programs and was specifically designed to simplify switching between different types of backends. The module provides three separate classes for each primitive type: 
+2. `BaseSampler` and `BaseEstimator`
 
-    1. `Sampler` and `Estimator`
-    These classes are reference implementations of both primitives and use Qiskit’s built-in simulator. They leverage Qiskit’s `quantum_info` module in the background, producing results based on ideal statevector simulations.
+These are abstract base classes that define a common interface for implementing primitives. All other classes in the `qiskit.primitives` module inherit from these base classes, and developers should use these if they are interested in developing their own primitives-based execution model for a specific backend provider. These classes may also be useful for those who want to do highly customized processing and find the existing primitives implementations too simple for their needs.
 
-    2. `BaseSampler` and `BaseEstimator`
-    These are abstract base classes that define a common interface for implementing primitives. All other classes in the `qiskit.primitives` module inherit from these base classes, and developers should use these if they are interested in developing their own primitives-based execution model for a specific backend provider. These classes may also be useful for those who want to do highly customized processing and find the existing primitives implementations too simple for their needs.
+3. `BackendSampler` and `BackendEstimator`
 
-    3. `BackendSampler` and `BackendEstimator`
-    If a provider does not support primitives natively, you can use these classes to “wrap” any backend into a primitive. Users can write primitive-style code for providers that don’t yet have a primitives-based interface. These classes can be used just like the regular Sampler and Estimator, except they should be initialized with an additional backend argument for selecting which backend to run on.
+If a provider does not support primitives natively, you can use these classes to “wrap” any backend into a primitive. Users can write primitive-style code for providers that don’t yet have a primitives-based interface. These classes can be used just like the regular Sampler and Estimator, except they should be initialized with an additional backend argument for selecting which backend to run on.
 
-* The primitives in ``qiskit_ibm_runtime`` provide access to cloud simulators and real hardware through the Qiskit Runtime service. They include exclusive features such as built-in circuit optimization and error mitigation support.
+The Qiskit Runtime primitives provide a more sophisticated implementation (such as with error mitigation) as a cloud-based service.
 
 ## Next steps
 
 <Admonition type="tip" title="Recommendations">
```

Comment:
Can you add a link to the "get started with primitives" page in the next steps section please?

Reply:
I set up a suggested addition for that link here https://github.com/Qiskit/documentation/pull/293/files#r1391373158